Tools/sitl_run.sh

```diff
@@ -121,6 +121,14 @@ elif [ "$program" == "gazebo" ] && [ ! -n "$no_sim" ]; then
 		echo "You need to have gazebo simulator installed!"
 		exit 1
 	fi
+elif [ "$program" == "flightgear" ] && [ ! -n "$no_sim" ]; then
+	echo "FG setup"
+    cd "${src_path}/Tools/flightgear_bridge/"
+    "${src_path}/Tools/flightgear_bridge/FG_run.py" ${model}.json    
+    sleep 20
```

Comment:
Do you really need to wait that long?

Reply:
Now, we have not any implementation of lock/check if FG is inicialized. And there is a problem, that, FG generic protocol sends bad data befere FG is fully inicialized.. 
So create this long sleep workround..

I will search for some check if FG is ready...

Reply:
Well, in last commit We solved this issue. There is no more sleep now and FG-bridge start send messages when simulation time is large than zero.

Reply:
Cool